manifests/freenas-12-stable.json

```diff
@@ -99,7 +99,6 @@
 				"NET_SNMP_WITHOUT_MIB_MODULE_LIST=sctp-mib",
 				"WANT_OPENLDAP_SASL=YES",
 				"WITHOUT_CHECK=YES",
-				"editors_jed_UNSET+=X11",
```

Comment:
We dont want X11

Reply:
We still unset it, it's just repeated twice in the manifest